src/clocksi_vnode.erl

```diff
@@ -150,15 +150,7 @@ get_active_txns_internal(TableName) ->
                     [] ->
```

Comment:
This case check can be left out completely, no?
`lists:flatmap` also works on the empty list `[]`, and we do not do anything special if the list has at least one element.

Reply:
Just committed a fix: https://github.com/AntidoteDB/antidote/pull/414/commits/cb6ce9fc54e26929a16250b8fc33a487e5905154